hw/formal/tools/csr_assert_gen/csr_assert_gen.core

```diff
@@ -5,7 +5,18 @@ CAPI=2:
 
 name: "lowrisc:fpv:csr_assert_gen"
 description: "Generator for CSR assertion check module used in FPV and DV testbenches."
-generators:
-  csr_assert_gen:
-    interpreter: python3
-    command: csr_assert_gen.py
+filesets:
+  files_dv:
+    depend:
+      # TODO(#5027, https://github.com/google/verible/issues/652):
+      # This is a hack to prevent the generated FPV assertion files to be
+      # passed into the Verible style linter. The tool currently does not
+      # support certain SVA language constrycts, causing it to throw syntax
+      # errors. Remove this once the issue has been addressed upstream.
+      - "! tool_veriblelint ? (lowrisc:fpv:csr_assert_gen_script)"
+      - "tool_veriblelint ? (lowrisc:fpv:csr_assert_gen_dummy)"
```

Comment:
this might be a bit controversial, so let me know what you think. 
the other option is to modify each DV corefile which I kind of tried to avoid since then the changes are spread all over.

Reply:
I think this looks like a good approach, especially as it should be temporary.